test/das.test.ts

```
@@ -259,6 +284,94 @@ test.serial('das: it can getAsset', async (t) => {
+    files: [{
+      mime: 'image/png',
+      uri: 'https://raw.githubusercontent.com/Bread-Heads-NFT/arcade-landing/refs/heads/main/public/asset.png',
+    }],
+    metadata: { name: 'Blockiosaurus', symbol: '' },
+    links: {
+      external_url: 'https://arcade.breadheads.io/',
+      image: 'https://raw.githubusercontent.com/Bread-Heads-NFT/arcade-landing/refs/heads/main/public/asset.png',
+    },
+  });
+
+  t.true(asset.appDatas!.length > 0, 'appDatas should not be empty');
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)